### PR TITLE
6502: Add support for Atari Tempest

### PR DESCRIPTION
**Pull request by mdeslaur**
Add support for Atari Tempest

### CODE REVIEW

File: libraries/C6502Cpu/CTempestBaseGame.cpp [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+#include "CTempestBaseGame.h"
+#include "C6502Cpu.h"
+#include <DFR_Key.h>
+
+//
+// Probe Head GND:
+//   6502 GND Pin 1 & GND Pin 21
+//
+// Watchdog Disable:
+//   Ground the WD DIS test point on the PCB near the CPU
```
Comment (prswan) [inline libraries/C6502Cpu/CTempestBaseGame.cpp:34]: 6502 compatibility can be problematic. I took a quick look at the Tempest schematics and it appears that the WRITE pulse is a simple NAND of CLK2, 3MHZ and RW that whilst unsynchronized with the Arduino has a good chance of working. Can you confirm that all the checks passed OK on the real PCB?

**Comment by mdeslaur** [comment]
Any chance you can take a look at this soon? I have a bunch of other Atari vector games I'll send next.

**Comment by Phillrb** [comment]
If Paul is busy or hasn't got a Tempest PCB to test this with, feel free to do a pull request on my fork: https://github.com/Phillrb/Arduino-ICT-PVAP
Asteroids, Asteroids DLX and Lunar Lander are already there (amongst others)

**Comment by prswan** [comment]
I left a comment but maybe it didn't get posted:
6502 compatibility can be problematic. I took a quick look at the Tempest schematics and it appears that the WRITE pulse is a simple NAND of CLK2, 3MHZ and RW that whilst unsynchronized with the Arduino has a good chance of working. Can you confirm that all the checks passed OK on the real PCB?

**Comment by mdeslaur** [comment]
Yes, I have used this code to debug a real Tempest PCB.

**Review by prswan — APPROVED** [review]

**Comment by prswan** [comment]
Thanks!
Paul.

**Comment by mdeslaur** [comment]
Thanks Paul!

**Comment by mdeslaur** [comment]
> If Paul is busy or hasn't got a Tempest PCB to test this with, feel free to do a pull request on my fork: https://github.com/Phillrb/Arduino-ICT-PVAP
> Asteroids, Asteroids DLX and Lunar Lander are already there (amongst others)

So what's the deal here with your fork, do you intend on merging it back at some point? Not knowing about it, I see that I've duplicated some work that's in it...

**Pull request merged**